builtin/logical/consul/path_roles.go

```diff
@@ -207,4 +225,6 @@ type roleConfig struct {
 	MaxTTL    time.Duration `json:"max_ttl"`
 	TokenType string        `json:"token_type"`
 	Local     bool          `json:"local"`
+	Namespace string        `json:"consul_namespace"`
```

Comment:
```suggestion
	ConsulNamespace string        `json:"consul_namespace"`
```

We should be consistent on the storage field and the JSON tag so we should either rename it to `ConsulNamespace` or change the marshaling to `namespace`.

Reply:
Makes sense, updated https://github.com/hashicorp/vault/pull/13850/commits/443e3e68fa3e880e3af23bbfee71692734df1395